kuegi_bot/bots/trading_bot.py

```diff
@@ -443,7 +487,7 @@ def save_open_positions(self, bars: List[Bar]):
             with open(base + self._get_pos_file(self.openPositionRolling), 'w') as file:
                 file.write(string)
             self.openPositionRolling += 1
-            if self.openPositionRolling > 10:
+            if self.openPositionRolling > 3:
```

Comment:
maybe have a constant somewhere, which is importable that everyone is able to define how much rolling position files they want to have?